### PR TITLE
Transfered ws from utils to saga and changed async component

### PR DESCRIPTION
**Pull request by Kaderimon**

### CODE REVIEW

**Review by RostyslavSt — APPROVED** [review]
+

File: src/index.js [hunk 1]
```diff
@@ -19,23 +19,22 @@ import { NotFound } from './components/NotFound/NotFound';
 import { config } from './config/config';
 import AsyncComponent from './components/AsyncComponent/AsyncComponent';
 import Register from './pages/Register/Register';
-import Login from './pages/Login/Login';
+
+// import Login from './pages/Login/Login';
```
Comment (totoshk) [inline src/index.js:23]: comment

File: src/index.js [hunk 2]
```diff
@@ -19,23 +19,22 @@ import { NotFound } from './components/NotFound/NotFound';
...
 const composeEnhancers = config.isProd ?
-  compose : window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__;
+  compose : window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose;
 const sagaMiddleware = createSagaMiddleware();
 const store = createStore(rootReducer, composeEnhancers(),
   applyMiddleware(sagaMiddleware));
 
 sagaMiddleware.run(rootSaga);
 
-/*
```
Comment (totoshk) [inline src/index.js:34]: comment

**Review by totoshk — APPROVED** [review]
Approve

**Pull request merged**